File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Cities/City.java
package com.appsbygu.kadai_weather_java.models.apis.Cities;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

import io.realm.RealmObject;

@Root(strict = false)
public class City extends RealmObject {

    @Attribute(name="id", required = false)
    private String id;

    @Attribute(name="title", required = false)
    private String title;


    public String getId ()
    {
        return id;
    }
    public String getTitle ()
    {
        return title;
    }
}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/cities/City.kt
package com.appsbygu.kadai_weather_java.models.apis.cities

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

import io.realm.RealmObject

@Root(strict = false)
open class City : RealmObject() {

    @field:Attribute(name = "id", required = false)
    var id: String? = null

    @field:Attribute(name = "title", required = false)
    var title: String? = null
}